tests/src/test/scala/system/basic/WskSequenceTests.scala

```diff
@@ -501,6 +501,46 @@ class WskSequenceTests extends TestHelpers with WskTestHelpers with StreamLoggin
     checkEchoSeqRuleResult(newRun, seqName, JsObject(newPayload))
   }
 
+  it should "run a sub-action even if it is updated while the sequence action is running" in withAssetCleaner(wskprops) {
```

Comment:
I've added a test case. And this test currently does not pass on the master branch.